Common/Deployment/DeploymentLib.ps1

```diff
@@ -821,7 +824,7 @@ function InitializeEnvironment()
         }
         if ($webResource -eq $null)
         {
-            if(HostEntryExists ("{0}.azurewebsites.net" -f $environmentName))
+            if(Test-AzureName -Website $environmentName)
```

Comment:
Thought we were using it from line 245 as well, but looks like we aren't.  Its interesting you aren't hitting this with storage?  Storage also uses HostEntryExists (line 285 for Storage resource type).